mri/operators/fourier/non_cartesian.py

```diff
@@ -574,11 +577,11 @@ def __init__(self, samples, shape, implementation='cpu', n_coils=1,
                 raise ValueError('gpuNUFFT library is not installed, '
                                  'please refer to README'
                                  'or use cpu for implementation')
-            self.implementation = gpuNUFFT(
+            self.impl = gpuNUFFT(
```

Comment:
Why not have this change for all the other implementations as well? Surely this will break something

Reply:
Well good point, this is what is breaking the tests xD . I am glad the tests caught this (and you too xD)